GOVERNANCE.md

```diff
@@ -165,6 +174,84 @@ We have a reasonably liberal approach to code review and merging. We value quick
+
+### Sponsorship
+
+Sponsorship is the first and foremost way of making financial contributions to Biome. Sponsorship can be
+done through a one-time donation, or through recurring donations. We offer incentives for certain levels of recurring donations.
+
+To learn more about sponsorship incentives, or to make a donation, please visit our [Open Collective page](https://opencollective.com/biome).
+You can also sponsor the project through [GitHub](https://github.com/sponsors/biomejs).
+
+### Community-Funded Bounties
+
+Bounties are the second approach towards funding Biome development. Unlike a sponsorship, bounties are directed towards the completion of a specific task. Tasks include, but are not limited to, the development of a feature or the fixing of a bug.
+
+Compared to sponsorship, bounties come with a few risks, both for Biome as a project, as well as the persons or entities offering and collecting the bounty. To minimize these risks, we only sanction bounties under the following conditions:
+
+- Bounties must be pledged through Polar.sh on an issue that is opened in our issue tracker.
+  - Unless otherwise specified, Biome reserves 30% of the payment amount, meaning contributors receive 70% of the pledged amount.
+  - The commission charged by Polar.sh is split accordingly.
+- If an issue in our issue tracker is not yet open for pledges, the person who wishes to pledge should indicate their desire in the comments of the issue, or do so through the `Community > #bounties` channel on Discord. A core contributor can then **accept** or **reject** the request for the bounty. Reasons for rejection include, but are not limited to: 
+  - Implementation of the task would conflict with the project direction or [its values](https://biomejs.dev/internals/philosophy/).
+  - Implementation of the task is (currently) technically infeasible.
+  - The suggested pledge is deemed inappropriate for the amount of work involved.
+- The person making the pledge acknowledges their pledge does not come with a guarantee of work done.
+- A task is only completed when a Biome maintainer merges the pull request that closes the task. The person trying to complete the task is responsible for addressing review comments to make sure the work meets the quality standards of the Biome project.
+- There is a 6 month expiration on pledges, as managed by Polar.sh.
+- Terms and conditions of Polar.sh apply to all pledges.
+  - For the FAQ, please see: https://docs.polar.sh/maintainers/faq/
+  - For the legal terms, please see: https://polar.sh/legal/terms
+- If you are receiving money as part of a bounty, please consider whether you need to pay taxes in your country.
+
+### Project-Funded Bounties
+
+In some cases, Biome funds bounties on its own issues. For project-funded bounties, the same conditions apply as for community-funded bounties. But given that these issues are funded by money from our sponsors, we have additional restrictions in place:
```

Comment:
> But given that these issues are funded by money from our sponsors, we have additional restrictions in place:

This isn't clear to me. Sponsors don't fund a "bounty", they fund Biome as a whole with some perks, but they fund particular "projects" or "issues". Maybe we should revisit this phrase.

Also, how does Biome funds itself? This needs to be stated, because this isn't clear to me.

Reply:
All our money comes from sponsors, so any time we fund issues through bounties, we're spending money received from our sponsors. I'll see if I can adjust the wording a bit.

Reply:
Actually, no. It isn't possible (nowadays) to fund issues via polar.sh using money coming from our open collective.

Reply:
Ah, that’s an important caveat indeed. Then I’ll just revise this to remain limited to core contributions for now.

Reply:
Done.